README.md

```diff
@@ -114,9 +114,9 @@ job.batch/volcano-admission-init   1/1           48s        96s
 
 You can reach the maintainers of this project at:
 
-Slack: [#volcano-sh](volcano-slack). (Signup [here](slack-invitation)).
```

Comment:
Simply remove # before volcano-sh

Reply:
And you can use inline link for `Signup [here]`

Reply:
@kevin-wangzefeng ,if we remove # before volcano-sh  then also link is not working.
I hope the below one would be fine

Slack: https://volcano-sh.slack.com. (Signup [here](https://join.slack.com/t/volcano-sh/shared_invite/enQtNTU5NTU3NDU0MTc4LTgzZTQ2MzViNTFmNDg1ZGUyMzcwNjgxZGQ1ZDdhOGE3Mzg1Y2NkZjk1MDJlZTZhZWU5MDg2MWJhMzI3Mjg3ZTk)).

Mailing List: https://groups.google.com/forum/#!forum/volcano-sh